specification/network/resource-manager/Microsoft.Network/stable/2018-12-01/webapplicationfirewall.json

```diff
@@ -0,0 +1,530 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-12-01",
+    "title": "WebApplicationFirewallManagement",
+    "description": "APIs to manage web application firewall rules"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
```

Comment:
Please add `security` property to fix CI errors
See https://github.com/Azure/azure-rest-api-specs/blob/master/specification/network/resource-manager/Microsoft.Network/stable/2018-12-01/endpointService.json#L18 for reference

Reply:
added